engine/sql/158_workflow_node_permission.sql

```diff
@@ -0,0 +1,14 @@
+-- +migrate Up
```

Comment:
Should we drop the foreign key for tables *_group to avoid problem if deleting an app, env... ?